pkg/config/config.go

```diff
@@ -56,6 +59,19 @@ type OrgOptConfig struct {
 	DisableRepoOverride bool `yaml:"disableRepoOverride"`
 }
 
+// OrgActionConfig is used in Allstar and policy-secific org-level config to
+// define the action configuration.
+type OrgActionConfig struct {
+	// IssueLabel : set to override GitHubIssueLabel in operator.go.
+	// GitHubIssueLabel is the label used to tag, search, and identify GitHub
+	// Issues created by the bot.
+	IssueLabel string `yaml:"issueLabel"`
+	
+	// IssueFooter : set to override GitHubIssueFooter in operator.go.
+	// IssueFooter is added to the end of GitHub issues.
+	IssueFooter string `yaml:"issueFooter"`
```

Comment:
The operator.go has default action config `GitHubIssueLabel` and `GitHubIssueFooter` so support both in this PR.